cost-analyzer/templates/cost-analyzer-frontend-config-map-template.yaml

```diff
@@ -109,17 +111,6 @@ data:
   {{- end }}
 
   {{- if and (not .Values.agent) (not .Values.cloudAgent) (not (eq (include "aggregator.deployMethod" .) "disabled")) }}
-    upstream aggregator {
-        {{- if .Values.kubecostFrontend.useDefaultFqdn }}
-        server {{ .Release.Name }}-aggregator.{{ .Release.Namespace }}.svc.cluster.local:9004;
-        {{- else }}
-        {{- if (.Values.kubecostFrontend.aggregator).fqdn }}
-        server {{ .Values.kubecostFrontend.aggregator.fqdn }};
-        {{- else }}
-        server {{ .Release.Name }}-aggregator.{{ .Release.Namespace }}:9004;
-        {{- end }}
-        {{- end }}
-    }
```

Comment:
Another minor thing for readability.

Kind of confusing that we are deleting this `upstream`, but keeping the `upstream model`? Everything routes to model which routes to aggregator? Would it be simpler if we try to use the `/aggregator` route everywhere in this config?

And then `upstream model` should just reroute to `/aggregator`?